app/Http/Controllers/DashboardController.php

```diff
@@ -23,100 +23,72 @@ class DashboardController extends Controller
      */
     public function index()
     {
-        $lastUpdatedContacts = Contact::where('account_id', Auth::user()->account_id)
-                                    ->orderBy('updated_at', 'desc')
-                                    ->limit(10)
-                                    ->get();
+        $account = Auth::user()->account;
 
-        // Latest statistics
-        $contacts = Contact::where('account_id', Auth::user()->account_id)
-                                    ->get();
+        $lastUpdatedContacts = $account->contacts->take(10);
```

Comment:
Suggestion: change this line to `$account->contacts()->limit(10)->get()` to reduce the data that has to be returned by the DB.